modules/apps/frontend-editor/frontend-editor-ckeditor-sample-web/src/main/resources/META-INF/resources/view_xss.jsp

```diff
@@ -0,0 +1,45 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<portlet:renderURL var="viewURL">
+	<portlet:param name="mvcPath" value="/view.jsp" />
+</portlet:renderURL>
+
+<a class="bg-light d-inline-flex my-2 p-2 rounded text-decoration-none" href="<%= viewURL %>">Back</a>
```

Comment:
Is is not possible to refresh the same page, implement the XSS case on the same legacy editor implementation, instead of going to the new page and new editor? There only needs to be a parameter set on refresh request, that triggers funky `aui:form:portletNamespace` attribute logic. 

The nice-ish buttons can be backup solution, if for some reason above is not possible.

Reply:
As this is an urgent security risk fix, this is not a blocker. We can improve our sample and the test in a followup. The only change really necessary is the `test.properties` one.